allennlp/training/deepspeed/__init__.py

```diff
@@ -1 +1,11 @@
-from allennlp.training.deepspeed.deepspeed_trainer import DeepspeedTrainer
\ No newline at end of file
+from allennlp.training.deepspeed.trainer import DeepspeedTrainer
```

Comment:
Todo before review:

1. Catch up to master
2. Implement checkpointing
3. Add LR scheduler, activation checkpointing, communication options to config

Stretch goals:
* Get sparse pretrained transformers working
* Pipeline parallelism